webpack/localization-plugin/README.md

```diff
@@ -0,0 +1,181 @@
+
+This option allows pseudolocales to be generated from the strings in the default locale. This option takes
+an option with pseudolocales as keys and options for the
+[pseudolocale package](https://www.npmjs.com/package/pseudolocale) as values.
+
+### `filesToIgnore = [ ]`
+
+This option is used to specify `.resx` and `.loc.json` files that should not be processed by this plugin.
+By default, every `.resx` and `.loc.json` file import is intercepted by this plugin, and an error occurs
+if translations aren't provided for an intercepted file. To avoid that error, list files that should be ignored
+by this plugin in this property. Files should be specified as either absolute paths or paths relative
+to the Webpack compilation context.
+
+### `localizationStats = { }`
+
+#### `localizationStats.dropPath = '...'`
+
+This option is used to designate a path at which a JSON file describing the localized assets produced should be
+written. If this property is omitted, the stats file won't be written.
+
+The file has the following format:
+
+```JSON
+{
+  "entrypoints": {
+    "<BUNDLE NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    },
+    "<BUNDLE NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    }
+  },
+  "namedChunkGroups": {
+    "<CHUNK NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    },
+    "<CHUNK NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    }
+  }
+}
+
+```
+
+#### `localizationStats.callback = (stats) => { ... }`
+
+This option is used to specify a callback to be called with the stats data that would be dropped at
+[`localizationStats.dropPath`](#localizationStats.DropPath--) after compilation completes.
+
+### `typingsOptions = { }`
+
+This option is used to specify how and if TypeScript typings should be generated for loc files.
+
+It takes two options:
+
+#### `typingsOptions.generatedTsFolder = '...'`
+
+This property specifies the folder in which `.d.ts` files for loc files should be dropped. It is recommended
+that this be a folder parallel to the source folder, specified in addition to the source folder in the
+[`rootDirs` `tsconfig.json` option](https://www.typescriptlang.org/docs/handbook/compiler-options.html).
+**The folder specified by this option is emptied when compilation is started.**
+
+This property is required if `typingsOptions` is set.
+
+#### `typingsOptions.sourceRoot = '...'`
+
+This optional property overrides the compiler context for discovery of localization files for which
+typings should be generated.
+
+### `typingsOptions.exportAsDefault = true | false`
+
+If this option is set to `true`, loc modules typings will be exported wrapped in a `default` property. This
```

Comment:
Haven't we deprecated `export default` with strong prejudice? What's the benefit of offering this feature?

Reply:
We haven't forced deprecation of `export default`, but we do discourage its usage. A few large repos inside Microsoft still import `.resx` files using `import x from './strings.resx';`, so we want to support this for now at least.